app/models/concerns/authorizable.rb

```diff
@@ -30,7 +30,6 @@
     invite_fosterers
     purge_attachments
     manage_default_pet_tasks
-    manage_fosters
```

Comment:
I removed the ManageFosterPolicy. Since fosters are actually just Match records and currently managing different types of Matches does not require different levels of permission, I think this is simpler for now.

If we want to differentiate them, I think we could build that into the MatchPolicy instead so that it checks permissions based on the `@match.match_type`.